src/backend/server.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
```

Comment:
GENERAL COMMENT
The ts code all looks to be fine. I would be using tsc to compile this into a build file now and trying to run it. See if you can do this, there's nothing to lose as you can always undo the compilation and it won't change your ts code

Reply:
i want to implement the tsc compiler by next mission, so i will look into that